### DISCUSSION TITLE
zebar restart. focus --next-workspace. focus --prev-workspace

### DISCUSSION THREAD

**Discussion by trunar** [glzr-io/glazewm#826] [Q&A]
I have been using an August release of glazewm for a couple of months now. Recently I decided to update because there have been some core changes to glazewm and zebar.

After updating I noticed that some things did not work the way they did before or the way I wanted them to work.

Firstly, when I restart config, I want glazewm to also kill the zebar process and start it again, before there was no problem with just one hotkey.
`config_reload_commands: ['shell-exec taskkill /IM zebar.exe /F', 'shell-exec zebar'].`
This parameter should kill the process after reloading a config and then start a new zebar process. For some reason this does not happen. It only runs the first command in the list, whatever it is. I tried to make it a command, tried to put it not in the reload commands but directly in the hotkey. The logic is the same, it only runs one of those two commands at a time. What I discovered is that if you put something between two commands that give you a warning or error in the shell, after you close the warning the second command will run as expected from the start.

The second thing is that in an August release I'm sure I used `focus --next-workspace` and `focus --prev-workspace` and they cycled within **already opened** workspaces, not all possible workspaces. Is there a flag or something that can change the behaviour of the commands back to the old style?

**Comment by trunar** [comment]
I also have another question. While in full screen mode in some games I still have a border around the full screen windows and rounded too. This has never happened before. Is there a fix for this?

**Comment by oligocene** [comment]
The next/prev commands where changed in [v.3.5.0](https://github.com/glzr-io/glazewm/releases/tag/v3.5.0), and two new ones added.

**Reply by trunar** [reply]
Thank you, that's exactly what I needed.

The questions about zebar and border in the full screen window are still relevant.
